sound/soc/sof/intel/hda-dai-ops.c

```diff
@@ -145,16 +145,9 @@ static struct hdac_ext_stream *hda_assign_hext_stream(struct snd_sof_dev *sdev,
 						      struct snd_soc_dai *cpu_dai,
 						      struct snd_pcm_substream *substream)
 {
-	struct snd_soc_pcm_runtime *rtd = snd_soc_substream_to_rtd(substream);
-	struct snd_soc_dai *dai;
 	struct hdac_ext_stream *hext_stream;
 
-	/* only allocate a stream_tag for the first DAI in the dailink */
-	dai = snd_soc_rtd_to_cpu(rtd, 0);
-	if (dai == cpu_dai)
-		hext_stream = hda_link_stream_assign(sof_to_bus(sdev), substream);
-	else
-		hext_stream = snd_soc_dai_get_dma_data(dai, substream);
+	hext_stream = hda_link_stream_assign(sof_to_bus(sdev), substream);
```

Comment:
the comment in the commit message is way too short, and it should say which commit is reverted by this change.

Reply:
comment still stands

Reply:
Oh, yet. I didn't notice that is a reversion.